### PR TITLE
Implement JWT in application

### LINKED ISSUE #105: Implement JWTs for application.

### PR DESCRIPTION
**Pull request by kdjohnson**
Closes #105

### CODE REVIEW

**Review by joshherkness — APPROVED** [review]

**Comment by joshherkness** [comment]
Waiting for other approvals before this gets merged in.  Good work @kdjohnson.

**Review by AnthonyHewins — APPROVED** [review]

**Pull request merged**
